docs/spec/distribution/03_end_block.md

```diff
@@ -1,35 +1,26 @@
 # End Block
 
-At each endblock, the fees received are allocated to the proposer, community fund,
-and global pool.  When the validator is the proposer of the round, that
-validator (and their delegators) receives between 1% and 5% of fee rewards, the
-reserve community tax is then charged, then the remainder is distributed
-proportionally by voting power to all bonded validators independent of whether
-they voted (social distribution). Note the social distribution is applied to
-proposer validator in addition to the proposer reward. 
-
-The amount of proposer reward is calculated from pre-commits Tendermint
-messages in order to incentivize validators to wait and include additional
-pre-commits in the block. All provision rewards are added to a provision reward
-pool which validator holds individually
-(`ValidatorDistribution.ProvisionsRewardPool`). 
+At each `EndBlock`, the fees received are transfered to the distribution `ModuleAccount` and allocated to the proposer, community fund, and global pool.  When the validator is the proposer of the round, that validator (and their delegators) receives between 1% and 5% of fee rewards, the reserve community tax is then charged, then the remainder is distributed proportionally by voting power to all bonded validators independent of whether they voted (social distribution). Note the social distribution is applied to proposer validator in addition to the proposer reward.
```

Comment:
Somewhat hard to digest...Are you saying the fees + inflation are transferred to the distribution `ModuleAccount` and then from the distribution `ModuleAccount` are allocated to the stakeholders?

Reply:
I'll rephase